.buildkite/pipelines/showcase/pipeline.rb

```diff
@@ -79,6 +79,15 @@
     end
   end
 
+  command do 
+    label "Matrix Step: {{matrix.os}} x {{matrix.arch}}"
+    command "echo 'Building for {{matrix.os}} on {{matrix.arch}}'"
+    matrix setup: {
+      os: ["linux", "macos", "windows"],
+      arch: ["amd64", "arm64"]
+    }
+  end
```

Comment:
```suggestion
  command do 
    label "Simple Matrix Step: {{matrix}}"
    command "echo 'Building for {{matrix}}'"
    matrix ["linux", "macos", "windows"]
  end

  command do 
    label "Advanced Matrix Step: {{matrix.os}} x {{matrix.arch}}"
    command "echo 'Building for {{matrix.os}} on {{matrix.arch}}'"
    matrix setup: {
      os: ["linux", "macos", "windows"],
      arch: ["amd64", "arm64"]
    }
  end
```

I think our current implementation works with [basic 1d matrix form](https://buildkite.com/docs/pipelines/configure/workflows/build-matrix) of:
```yaml
steps:
  - label: "{{matrix}} build"
    command: "GOOS={{matrix}} go build"
    env:
      os: "{{matrix}}"
    matrix:
      - "darwin"
      - "Linux"
      - "Windows"
```
so adding this to the showcase as well.